package.json

```diff
@@ -16,11 +16,11 @@
     "create_db": "cross-env NODE_ENV=test node dist/create_db.js",
     "delete_db": "cross-env NODE_ENV=test node dist/delete_db.js",
     "build": "gulp",
-    "lint": "esw -c .eslintrc *.js server config . --color --fix",
+    "lint": "esw -c .eslintrc --color --fix .",
     "lint:watch": "yarn lint -- --watch",
     "beautify": "esw -c .eslintrc *.js server config --color --fix",
     "precommit": "yarn lint && yarn test",
-    "test": "gulp && yarn create_db && cross-env NODE_ENV=test ./node_modules/.bin/mocha --ui bdd --reporter spec --colors --compilers js:babel-core/register server/tests --recursive || yarn delete_db",
+    "test": "gulp && yarn create_db && cross-env NODE_ENV=test ./node_modules/.bin/mocha --ui bdd --reporter spec --colors --compilers js:babel-core/register server/tests --recursive; yarn delete_db",
```

Comment:
This caused an issue on my windows machine, why was this changed, I assume it has something to do with the mac terminal ?

Reply:
Forgot I messed with that. The problem I had was the database wasn't being deleted when the tests passed due to the `||`. But if the semicolon doesn't work on windows, I'm not sure what to do.

maybe something like:

```
gulp && yarn create_db && cross-env NODE_ENV=test ./node_modules/.bin/mocha --ui bdd --reporter spec --colors --compilers js:babel-core/register server/tests --recursive && yarn delete_db || yarn delete_db
```

?